flocker/node/_deploy.py

```diff
@@ -757,23 +757,22 @@ def calculate_changes(self, desired_configuration, current_cluster_state):
         if desired_open_ports != set(self.network.enumerate_open_ports()):
             phases.append(OpenPorts(ports=desired_open_ports))
 
-        current_node_applications = set(
+        running_applications = set(
             app for app in current_node_state.applications if app.running)
         all_applications = current_node_state.applications
 
         # Compare the applications being changed by name only.  Other
         # configuration changes aren't important at this point.
-        current_state = {app.name for app in current_node_applications}
+        running = {app.name for app in running_applications}
         desired_local_state = {app.name for app in
                                desired_node_applications}
         not_running = {
             app.name for app
-            in all_applications.difference(current_node_applications)}
+            in all_applications.difference(running_applications)}
 
         # Don't start applications that exist on this node but aren't
-        # running; instead they should be restarted:
-        start_names = desired_local_state.difference(
-            current_state | not_running)
+        # running; Docker is in charge of restarts:
+        start_names = desired_local_state.difference(running | not_running)
```

Comment:
Is there any reason to have both `running` and `not_running` here? It looks like this is the only usage of them.